packages/ckeditor5-list/tests/documentlistproperties/documentliststylecommand.js

```diff
@@ -0,0 +1,469 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import { setData, getData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
+import VirtualTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/virtualtesteditor';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import DocumentListPropertiesEditing from '../../src/documentlistproperties/documentlistpropertiesediting';
+import { modelList } from '../documentlist/_utils/utils';
+import stubUid from '../documentlist/_utils/uid';
+
+describe( 'DocumentListStyleCommand', () => {
+	let editor, model, bulletedListCommand, numberedListCommand, listStyleCommand, stub;
+
+	beforeEach( async () => {
+		const newEditor = await VirtualTestEditor.create( {
+			plugins: [ Paragraph, DocumentListPropertiesEditing ]
+		} );
```

Comment:
You should not use the full editor instance to test commands. We are usually instantiating only required classes to test commands to make sure that the command itself is generating a correct model state. Otherwise, post-fixers would fix the model and we would not know if the command itself is generating the correct output.